src/ui/components/DevTools.tsx

```diff
@@ -216,6 +219,27 @@ function _DevTools({
     }
   }, [loadingFinished, trackLoadingIdleTime, sessionId]);
 
+  useEffect(() => {
+    if (!userLoading && recording) {
+      setDefaultTags({
+        recording: {
+          id: recording.id,
+          title: recording.title,
+          url: recording.url,
+          userId: recording.user?.id,
+          workspace: recording?.workspace,
+          metadata: recording.metadata && {
+            testEnvironment: recording.metadata?.test?.environment,
+          },
+        },
+        user: {
+          id: userId,
+          email: userEmail,
+        },
```

Comment:
nit: `user` is kind of ambiguous. (It confused me to see `recording.user` and `user` at first glance.)

Maybe we could name this `session` instead?

```suggestion
        session: {
          userId,
          userEmail,
        },
```